api/v2/clairpb/clair.proto

```diff
@@ -26,57 +27,63 @@ message Vulnerability {
 	string link = 4;
 	string severity = 5;
 	string metadata = 6;
+	// fixed_by only exists when querying features
 	string fixed_by = 7;
-	repeated Feature fixed_in_features = 8;
+	// affected_versions only exists when querying notifications
+	repeated Feature affected_versions = 8;
 }
 
-message Feature {
+message ClairStatus {
+	// all listers and detectors implemented in this clair version
```

Comment:
Nope, we can do that later, just as long as nothing here prevents us from doing so.